### ISSUE TITLE
Lua language support

### ISSUE THREAD

**Issue by mika76** [dotnet/interactive#2548]
I know this might be a long shot, but polyglot notebooks are really useful, and I use lua for game programming, so it would be a match made in heaven to be able to use lua as a language in the notebooks. Any chance of it ever happening?

**Comment by colombod** [comment]
This is a very nice question. This is something that could be started as extension and plugin. having lua support is one thing, how would you envision the interaction with the game development?

**Comment by ParadiseFallen** [comment]
+1 on this issue.

We can fork moonsharp and upgrade it to 5.4LUA

Or just use native bindings

**Comment by dt192** [comment]
Yes, Lua seems to be the obvious one missing from the list of supported languages.
